src/components/Widget/components/Launcher/index.js

```diff
@@ -25,13 +28,48 @@ const Launcher = ({
   displayUnreadCount,
   showTooltip,
   lastMessage,
-  closeTooltip
+  closeTooltip,
+  lastUserMessage,
+  domHighlight
 }) => {
+  const [referenceElement, setReferenceElement] = useState(null);
+  useEffect(() => {
+    const setReference = (selector) => {
+      const reference = document.querySelectorAll(selector);
+      if (reference && reference.length === 1) {
+        onRemove(reference[0], () => setReferenceElement(null));
+        setReferenceElement(reference[0]);
+      } else {
+        setReferenceElement(null);
+      }
+    };
+    if (lastUserMessage && lastUserMessage.get('nextMessageIsTooltip')) {
+      setReference(lastUserMessage.get('nextMessageIsTooltip'));
+    } else if (domHighlight && domHighlight.get('selector')) {
+      setReference(domHighlight.get('selector'));
+    } else {
+      setReferenceElement(null);
+    }
+  }, [lastUserMessage, domHighlight]);
+  const [popperElement, setPopperElement] = useState(null);
+  const [arrowElement, setArrowElement] = useState(null);
+  const { styles, attributes } = usePopper(referenceElement, popperElement, {
+    modifiers: [
+      // The arrow padding ensures it never get on the border where it looks ugly
+      { name: 'arrow', options: { element: arrowElement, padding: 5 } },
+      {
+        name: 'preventOverflow',
+        options: {
+          padding: 15 // 0 by default
+        }
+      }
+    ],
+    placement: 'right'
```

Comment:
Minor: if the highlighted element id near the right border of the window, we won't be able to see the tooltip